app/queries/patient_states/diabetes/cumulative_assigned_patients_query.rb

```diff
@@ -0,0 +1,18 @@
+class PatientStates::Diabetes::CumulativeAssignedPatientsQuery
+  attr_reader :region, :period
+
+  def initialize(region, period)
+    @region = region
+    @period = period
+  end
+
+  def call
+    Reports::PatientState
+      .where(
+        assigned_facility_id: region.facility_ids,
+        month_date: period
+      )
+      .where(diabetes: "yes")
+      .where.not(htn_care_state: "dead")
```

Comment:
Yup, this is some tech debt. `htn_care_state` should have been called `care_state` since it's not dependent on the patients condition. But rename it would mean having to rename it in all the metabase reports also. 

Maybe we can create an duplicate column with the name `care_state` in future.